sp800-63b/sec4_aal.md

```diff
@@ -130,8 +130,9 @@ Authentication Assurance Level 3 requires the use of one of three kinds of hardw
  * Single-Factor Cryptographic Device
 * Multi-Factor Cryptographic Device
 * Single-Factor Cryptographic Device used in conjunction with Memorized Secret
+* Multi-Factor OTP Device used in conjunction with a single-factor cryptographic device
```

Comment:
The second "have" is needed because the MF OTP device isn't verifier impersonation resistant, so we need another authenticator that is. Another way to describe it might be "Single-Factor Cryptographic Device used in conjunction with either Memorized Secret or Multi-Factor OTP Device"

I notice that there's a rendering problem with the sub-bullets at line 129 and 130 so will be committing a fix for that.

I was afraid that one of the pull requests would result in a conflict on the other, so if this looks OK I can do the conflict resolution and merge if you want.